lib/dependencies/ImportParserPlugin.js

```diff
@@ -254,7 +254,7 @@ class ImportParserPlugin {
 				dep.loc = expr.loc;
 				dep.optional = !!parser.scope.inTry;
 				parser.state.current.addDependency(dep);
-				return true;
+				return false;
```

Comment:
I think walking the full expression is not correct as it would handle constant pieces twice.

like this one: ``import(`./${CONSTANT}/${DEFINED_EXPRESSION}`)``

with `DefinePlugin({ CONSTANT: JSON.stringify("folder"), DEFINED_EXPRESSION: "foobar" })`

Could you add this to the test?

Reply:
Sadly you are right. Seems only reliable way to fix this is to walk necessary expressions in `ContextDependencyHelpers`. I'll push a new iteration